src/Symfony/Component/Security/Core/Event/VoteEvent.php

```diff
@@ -28,6 +28,7 @@ public function __construct(
         private mixed $subject,
         private array $attributes,
         private int $vote,
+        private array $reasons = [],
```

Comment:
perhaps typehint `string[]` to give context?

Reply:
this class is internal